index.ts

```diff
@@ -1,54 +1,31 @@
 import { multiUrlGenerator } from './lib/multiUrlGenerator';
-import readline from 'readline';
+import fetch from 'node-fetch';
```

Comment:
As discussed offline, I removed `request` in favor of `fetch-node` sine `request` has been deprecated.